presto-hive/src/test/sql/create-test.sql

```diff
@@ -232,3 +232,8 @@ ALTER TABLE presto_test_partition_schema_change REPLACE COLUMNS (t_data DOUBLE);
 
 INSERT OVERWRITE TABLE presto_test_partition_schema_change_non_canonical PARTITION (t_boolean='0')
 SELECT 'test' FROM presto_test_sequence LIMIT 100;
+
+ANALYZE TABLE presto_test_unpartitioned COMPUTE STATISTICS;
+ANALYZE TABLE presto_test_unpartitioned COMPUTE STATISTICS FOR COLUMNS;
+ANALYZE TABLE presto_test_bucketed_by_string_int PARTITION(ds) COMPUTE STATISTICS;
+ANALYZE TABLE presto_test_bucketed_by_string_int PARTITION(ds) COMPUTE STATISTICS FOR COLUMNS;
```

Comment:
As a part of the "HIVE_TESTS=true" job in Travis, computing stats for the new tables seems to take a total of 71 seconds wall time, which for two tables with 100 rows each is shockingly slow IMO, but hey, it's hadoop.  I don't know of any reason why it would cause failures, but also I don't know of any reason why it would take more than a faction of a second.  I haven't seen any failure of the stats gathering in Travis or DUPA runs as I've worked on the test.

https://s3.amazonaws.com/archive.travis-ci.org/jobs/261345970/log.txt?X-Amz-Expires=29&X-Amz-Date=20170807T153032Z&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJRYRXRSVGNKPKO5A/20170807/us-east-1/s3/aws4_request&X-Amz-SignedHeaders=host&X-Amz-Signature=d8a068cbfdd0c7c86f8363bb40b102b0ef9fb92ccc5f01e3b1c85c0e4704f98e